pyproject.toml

```diff
@@ -31,12 +31,7 @@ dependencies = [
     "uvicorn[standard]>=0.24.0",
     "dspy>=2.6.27",
     "kubernetes>=33.1.0",
-]
-
-[project.optional-dependencies]
-crewai = [
-    "crewai",
-    "litellm==1.67.5"
+    "ddgs>=9.4.0",
```

Comment:
separate from my other comment, shouldn't this replace `duckduckgo-search` rather than be added as a new line?